src/actions/index.js

```diff
@@ -9,10 +9,12 @@ export const DECREMENT = 'DECREMENT';
 // logic of updating the central Redux store. That
 // is left to the reducer(s).
 
-export const increment = () => {
-  // Fill in this function
+export const increment = (number) => {
+  
+  return { type: 'INCREMENT', payload: number };
```

Comment:
This works, but convention is to use the variables defined at the top of this file, as opposed to strings. So INCREMENT instead of 'INCREMENT'. Notice how we export these variables here in action, and in the reducer how we import them? This allows us to use the variables in the switch statement in the reducer as well. We're not actually passing a `payload` here, so that part is unnecessary. This project differs from the lecture demo in that we're not dealing with handling any input.